che-theia-java-extension/package.json

```diff
@@ -12,7 +12,8 @@
     "@theia/core": "0.3.13",
     "@theia/java": "0.3.13",
     "@theia/editor": "0.3.13",
-    "@theia/languages": "0.3.13"
+    "@theia/languages": "0.3.13",
```

Comment:
should be bumped to 0.3.15 now (to see if it's working fine with latest of Theia)